Additional update/edits to CHANGELOG.md for 1.1 (#1791)
SynthstromAudible/DelugeFirmware: CHANGELOG.md

- Added a menu for Song-level parameters, accessible in `SONG VIEW` and `ARRANGER VIEW` by pressing `SELECT`.
- Added a `KIT FX MENU` to KITS which allows you to customize the AFFECT ENTIRE KIT parameters with greater control.   Accessible in `KIT CLIP VIEW` by pressing `SELECT` with `AFFECT ENTIRE` enabled.
- Added support for LUMI Keys SYSEX protocol. When hosting a LUMI Keys keyboard, the Deluge's current scale will
  automatically be set on the LUMI's keyboard.
- Streamlined recording new clips in `GRID VIEW` while the Deluge is playing. Short-press empty clip pads in Grid View's `GREEN MODE` while recording is armed to automatically create a new clip and queue it to record at the start of the next bar.
- Fixed a bug preventing clip selection while `SHIFT` was held.
- Fixed numerous bugs, including some crash bugs, around the display of `QUANTIZED STUTTER`.
- Fixed a bug with `SHIFT + SCROLL SELECT ENCODER` on small menus which would allow moving off the end of the menu, causing crashes.
- Fixed several bugs with pad grid rendering.
- Added Master Chromatic Transpose of All Scale Mode Instrument Clips from any SONG View (SONG, ARRANGER,

>   ARRANGER AUTOMATION and PERFORMANCE VIEW.) Uses the same shortcut as in a Synth/Midi/CV clip (Press and

Reviewer: Change `Arranger Automation` to `Arranger Automation View` please! thx

  turn `▼︎▲︎` to transpose all clips by +/- 1 semitone). The number of semitones transposed is customizable (Press `SHIFT`
  + turn `▼︎▲︎`). After transposing, the display shows the new Root Note (and Scale Name on OLED displays).

> - Added `SIDEBAR CONTROLS` in  `KEYBOARD VIEW` for synths. By default the two sidebar columns pertain to `VELOCITY` (red) and `MOD WHEEL` (blue). Holding a pad sets it momentarily to that value, tapping a pad latches it to that value. The functionality can be changed per column by holding the top pad and turning `SELECT` and include 'CHORDS', `CHORD MEMORY`, and `SCALES`.

Reviewer 2: ```suggestion
- Added `SIDEBAR CONTROLS` in  `KEYBOARD VIEW` for synths. By default the two sidebar columns pertain to `VELOCITY` (red) and `MOD WHEEL` (blue). Holding a pad sets it momentarily to that value, tapping a pad latches it to that value. The functionality can be changed per column by holding the top pad and turning `▼▲`. Available options are `VELOCITY`, `MOD WHEEL`, 'CHORDS', `CHORD MEMORY`, and `SCALES`.
```

> - Added a `VU METER` toggle that displays the VU Meter on the sidebar in `SONG`, `ARRANGER`, & `PERFORMANCE VIEW`.

Reviewer 2: ```suggestion
- Added a `VU METER` toggle that displays the VU Meter on the sidebar in `SONG`, `ARRANGER`, & `PERFORMANCE VIEW`. To activate it, on a song view ensure `AFFECT-ENTIRE` is enabled, then select the `LEVEL/PAN` mod button and press it again to activate the meter.
```

- Added new `High CPU Usage Indicator`. The `PLAY` button button will blink when deluge CPU usage is high which indicates that synth voices / sample playback may be culled.
  - To activate the feature, press `SHIFT` + `SELECT` : `MENU > DEFAULTS > HIGH CPU INDICATOR`.
- Removed ability to convert an Audio Clip to an Instrument Clip (Synth/Kit/MIDI/CV) as this conversion process is currently error/bug prone.
- Restricted changing Synth/MIDI/CV Clips to the Kit Clips and vice versa if the clip is not empty.
- Added retrigger to all keyboard views.
- Added a new default setting that controls which playback mode new slices of a kit will receive. 
- Added indicators when `WAVEFORM LOOP LOCK` is enabled. 7SEG displays a rightmost `.` on screen and OLED displays a lock icon.
- Modified waveform marker rendering to improve clarity.
- Created a new menu hierarchies document that documents the Deluge menu structure for OLED and 7SEG and can be used as a reference for navigating the various menu's. See: [Menu Hierarchies](https://github.com/SynthstromAudible/DelugeFirmware/blob/community/docs/menu_hierarchies.md)

> - Added MIDI learning of Song Params.

Reviewer 2: ```suggestion
```
This has been removed for now.

- Added Synth/MIDI/CV clip configuration of note row play direction. Hold `AUDITION` while entering the `PLAY DIRECTION` menu to set the play direction for the selected note row. While in the note row play direction menu, you can select other note rows to quickly set the play directiom for multiple note rows.

In addition, a number of improvements have been made to how the OLED display is used:

- Added parameter names (including mod matrix / patch cable mappings) to Mod (Gold) Encoder popups.
- `ARRANGER VIEW` and `SONG VIEW` now display the name of the current view on the screen.
- The 12TET note name is now displayed along with the MIDI note number.

> - Added a new community setting which allows emulating the 7SEG style on the OLED display. When set to 'TOGGLE' press `SHIFT`+`LEARN/INPUT`+`AFFECT ENTIRE` to switch to the emulated 7SEG display. 

Reviewer 2: ```suggestion
- Added a new community setting which allows emulating the 7SEG style on the OLED display. When set to `TOGGLE` press `SHIFT`+`LEARN/INPUT`+`AFFECT ENTIRE` to switch to the emulated 7SEG display. 
```

- Fixed several cases where popups could get stuck open.
- Fixed a number of minor rendering bugs.

### Sequencer

- When changing instrument presets in `SONG VIEW` or `ARRANGER VIEW`, all pertaining clips of that instrument switch as well. Individual clips can still have only their preset changed by doing so in `CLIP VIEW`.
- Added support for 5 and 6 note scales.
- Added 8 new built-in scales: Melodic Minor, Hungarian Minor, Marva (Indian), Arabian, Whole Tone, Blues, Pentatonic
  Minor, Hirajoshi.

> - Added `ONCE CLIP` launch mode, which allows a clip to play just once when launched and then mute itself. Settable in Song's `ROWS VIEW` by holding the `MUTE` pad of a row and then pressing `SELECT`. Causes 

Reviewer 2: ```suggestion
- Added `ONCE CLIP` launch mode, which allows a clip to play just once when launched and then mute itself. Settable in Song's `ROWS VIEW` by holding the `MUTE` pad of a row and then pressing `SELECT`. 
```
